dbms/src/Storages/DeltaMerge/DeltaMergeStore.cpp

```diff
@@ -369,12 +369,8 @@ void DeltaMergeStore::rename(String /*new_path*/, bool clean_rename, String new_
     db_name.swap(new_database_name);
 }
 
-void DeltaMergeStore::drop()
+void DeltaMergeStore::dropAllSegments(bool keep_first_segment)
```

Comment:
I think we can add some failpoints to help testing whether the restoring and removing data perform as expected.

Reply:
Added.